ArduPlane/Attitude.cpp

```diff
@@ -370,15 +389,17 @@ void Plane::stabilize()
         const float elevator = pitchController.get_rate_out(nav_scripting.pitch_rate_dps, speed_scaler);
         SRV_Channels::set_output_scaled(SRV_Channel::k_aileron, aileron);
         SRV_Channels::set_output_scaled(SRV_Channel::k_elevator, elevator);
+        float rudder = 0;
         if (yawController.rate_control_enabled()) {
-            float rudder = nav_scripting.rudder_offset_pct * 45;
+            rudder = nav_scripting.rudder_offset_pct * 45;
```

Comment:
Scripting stall turns are unchanged as it set this `rudder_offset_pct` which work as before.